assets/css/style.css

```diff
@@ -65,47 +128,49 @@
     border-radius: 10px; 
 }
 
-.map-area {
-    border-radius: 10px;
+#itinerary-list {
+    background-color: var(--inner-color);
 }
 
-.hide {
-    display:none;
+#itinerary-title {
+    background-color: #ffbe76;
+    color: white;
+    font-weight: 400;
+    font-size: 20px;
 }
 
-.invalid-search {
-    border: solid red;
-    display: block;
```

Comment:
Won't this remove the red error handling for invalid city?

Reply:
This code is still in there, I just moved it

Reply:
sorry I didn't catch it earlier